goblin_hunt.py

```diff
@@ -0,0 +1,18 @@
+def goblin_hunt():
+    print("Welcome to the Angry Goblin Hunt")
+    print("n award-winning game full of adventure and excitement (!)")
+    input("Type in your name:")
+    print("Stephen, do you think you can find the goblin hiding in the kitchen cupboards?")
```

Comment:
This needs to be correctd.

<img width="1011" alt="image" src="https://user-images.githubusercontent.com/38807024/201337051-7554e9a0-3680-42e3-9ea0-da6c67ab2009.png">